HotSpot/View Controllers/DetailsViewController.m

```diff
@@ -44,8 +46,22 @@ - (void)viewDidLoad {
         
         self.homeownerNumber = object[@"phone"];
     }];
-    
-    
+}
+
+- (IBAction)reportHomeowner:(UIButton *)sender {
+    UIAlertController *errorAlert = [[UIAlertController alloc] init];
```

Comment:
would be nicer to have a title and message as well. you can use this init method https://developer.apple.com/documentation/uikit/uialertcontroller/1620092-alertcontrollerwithtitle?language=objc